### DISCUSSION TITLE
Invalid copilot token: missing token: 403

### DISCUSSION THREAD

**Discussion by selfrefactor** [community/community#40019] [Copilot Conversations] [answered]
### Select Topic Area

Bug

### Body

I tried all suggested steps in relevant issues including revoke of VSCode Github application. I was in beta and now I am in free trial mode. No need to share that this is not making me want to continue:

```
[INFO] [auth] [2022-11-24T21:54:57.296Z] Invalid copilot token: missing token: 403 
[ERROR] [default] [2022-11-24T21:54:57.298Z] GitHub Copilot could not connect to server. Extension activation failed: "Thank you for participating in the GitHub Copilot Technical Preview. Your extended access has ended."
```

**Comment by loops-and-spells** [comment]
Same. Is there an outage?

**Comment by Baivo** [comment]
Hello from google!
Also experiencing similar.

[INFO] [auth] [2022-11-24T23:44:57.714Z] Invalid copilot token: missing token: 403 Forbidden
[ERROR] [default] [2022-11-24T23:44:57.717Z] GitHub Copilot could not connect to server. Extension activation failed: "No access to GitHub Copilot found."

**Comment by crierr** [comment]
Same here. I'm subscribing monthly plan

[INFO] [auth] [2022-11-25T01:52:35.144Z] Invalid copilot token: missing token: 403 
[ERROR] [default] [2022-11-25T01:52:35.145Z] GitHub Copilot could not connect to server. Extension activation failed: "Thank you for participating in the GitHub Copilot Technical Preview. Your extended access has ended."

**Comment by GGICE** [comment]
Same here. I'm subscribing year plan

[INFO] [auth] [2022-11-25T02:03:30.552Z] Invalid copilot token: missing token: 403 
[ERROR] [default] [2022-11-25T02:03:30.552Z] GitHub Copilot could not connect to server. Extension activation failed: "Thank you for participating in the GitHub Copilot Technical Preview. Your extended access has ended."

**Reply by aryairani** [reply]
Same.

**Reply by ravhirizaldi** [reply]
Same

**Comment by RyanZurrin** [comment]
Having the same issue: Is there a fix to this? I am assuming that there should be a way for us to update our tokens.

**Reply by RyanZurrin** [reply]
It was working fine for me yesterday and even earlier today but then today all of a sudden stopped working across all my devices.  I hope there is a fix soon.

**Comment by DaemonStark** [comment]
I am having the same issue as well, is there an outage? Please introduce a way to update our tokens / authorization details.

**Comment by hieunh1801** [comment]
I have the same issuse

**Comment by MarianoDAcosta** [comment]
We are a bunch of people facing the same issue, so I assume there should be a problem with the server. I have already paid the $10 for a month and a couple of days ago GitHub has tried to recharge the $10 dollars again after 20 days of the last payment and today it suddenly stopped working, with a 403 error, GitHub Copilot could not connect to server. Extension activation failed: "Thank you for participating in the GitHub Copilot Technical Preview. Your extended access has ended."
I have already claimed this to GitHub, by opening a ticked.

**Reply by Maxim-Filimonov** [reply]
Do you have a ticket link?

**Reply by MarianoDAcosta** [reply]
Yes, sure:
https://support.github.com/ticket/1892192

**Comment by lozzo** [comment]
I have the same issuse, but my colleagues can use copilot at the same time

**Comment by adityarizqi** [comment]
i have same issue, cc @github-staff

**Comment by xesrevinu** [comment]
i have same issue same issuse,

- sign out & sign in ❌
- reinstall ❌

Copilot Labs It looks the same problem.
`GitHub Copilot Labs could not connect to server. Extension activation failed: "No instance of  has been registered.`

**Comment by indrex** [comment]
It is not just me! Fix it guys!

**Comment by jasperdunn** [comment]
Found this on your ticket @MarianoDAcosta

"Copilot is currently status yellow; with an update as of 39 minutes ago. This may affect Copilot behavior and performance."
https://www.githubstatus.com/incidents/69flmdbxhtsk

**Comment by c-conklin** [comment]
Of course, I see this after submitting a ticket. Also experiencing the issue as a yearly subscriber. Looks like they're aware of the status!

**Comment by davecheney** [comment, marked as answer]
Hello. I'm sorry you had difficulty accessing Copilot, please accept my apologies on behalf of the Copilot team. This was caused by a bad deploy which affected a small percentage of Copilot users. The change has been rolled back and you should be able to use Copilot again.

**Reply by compil3** [reply]
still not working as of 8:21pm.

**Comment by selfrefactor** [comment]
Yes, I can confirm that it works now. Good speed of resolution.

**Comment by joshuafuller** [comment]
Just an update for anyone currently having this issue and landing here.

It looks like the problem is still ongoing.
Status Page link for Incident: https://www.githubstatus.com/incidents/3208bb8blz5k

**Reply by aryairani** [reply]
Same, yes, still having the issue; logging out and in again in VS Code has no effect

**Reply by aryairani** [reply]
@davecheney Thanks in advance

**Reply by joshuafuller** [reply]
Same; I also tried removing and re-adding the CoPilot extension with no success. 

Guess I'll take advantage of this and get some fresh air.
I ❤️ CoPilot

**Reply by aryairani** [reply]
It's working again for me this morning.

**Comment by jeremiahsherrill** [comment]
> We are investigating reports of degraded performance for Copilot.

lol -- not working is not the same as degraded performance.
